digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/process/RoadAddressChangeInfoMapperSpec.scala

```diff
@@ -102,6 +104,7 @@ class RoadAddressChangeInfoMapperSpec extends FunSuite with Matchers {
     results(roadLinkId3).size should be (1)
     results(roadLinkId3).count(_.id == -1000) should be (0)
     results(roadLinkId3).head.eq(roadAddress3) should be (true)
+    results.values.flatten.map(_.commonHistoryId).toSet.size should be (1)
```

Comment:
I think we should check if it still have the same value after the process execute. Not only check if the value is equal. Because let's imagine that we generate a new common history id for all, the test will pass but and it's not suppose.

Reply:
In that cases we would probably have rows with old commonHistoryIds and rows with newCommonHistoryIds that would make the set to have 2 different commonHistoryId results. But ive still made the change as it would make the code more readable.